summingbird-storm/src/main/scala/com/twitter/summingbird/storm/EdgeGrouping.scala

```diff
@@ -0,0 +1,26 @@
+package com.twitter.summingbird.storm
+
+import org.apache.storm.topology.BoltDeclarer
+import org.apache.storm.tuple.{ Fields => StormFields }
+
+/**
+  * This trait is used to represent different grouping strategies in `Storm`.
+  */
+sealed trait EdgeGrouping {
+  def apply(declarer: BoltDeclarer, parentName: String): Unit
```

Comment:
Can you comment what this does? Like what do the implementors of this trait need to do to confirm with the protocol.